kani-driver/src/call_cbmc.rs

```diff
@@ -145,27 +145,28 @@ impl KaniSession {
             .await)
         };
 
-        let verification_results = if res.is_err() {
+        if let Ok(output) = res {
+            // The timeout wasn't reached
+            Ok(VerificationResult::from(
+                output.unwrap(),
```

Comment:
Thanks for the refactoring. If we want to maintain the same behavior, this should be:
```suggestion
                output?,
```